### ISSUE TITLE
Accessibility Feedback - Design Intent - [Auth Exp My VA, Benefit Payments and Debt version 2]

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#40699]
## VFS acceptance criteria
- [x] Let Platform know when the **Must** feedback has been incorporated 
- [x] Leave any comments for feedback you decide _not_ to take
- [ ] VFS team closes the ticket

## Thoughts/questions
- Just to reiterate what everyone was saying in the meeting: This was great, especially for your first time through a Design Intent! I particularly appreciate you bringing up accessibility concerns proactively.

## Feedback
- **Must** = the feedback must be applied
- **Should** = the feedback is best practice
- **Consider** = suggestions/enhancements

**Must:**
After the meeting I checked color contrasts. `green`  #2e8540 against `gray-lightest` #f1f1f1 has a [contrast ratio of 4.09:1](https://webaim.org/resources/contrastchecker/?fcolor=2E8540&bcolor=F1F1F1). 

Greens are notoriously tough. The [WCAG 2.1 AA-level guidelines for minimum contrast](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) are:
* 4.5:1 ratio or better for normal text
* 3:1 ratio or better for large text, where "large" is defined as
    - 18pt (approx 24px) or larger
    - Bold and 14pt (approx 18.5px) or larger

In the Sketch file, it looks like the green text is 16px, so you're just below the line. It looks like `green-darker` #195C27 would have a contrast ratio of 7.14:1. I would recommend you go with that --- or explore other shades between green and green-darker in consultation with the Design System team.

Note: Even with a color change here, I would still recommend testing with colorblind users as well.

**Should:**

**Consider:**
- I also checked the red contrast combination. `secondary-dark` #cd2026 against `gray-lightest` #f1f1f1 has a [contrast ratio of 4.85:1](https://webaim.org/resources/contrastchecker/?fcolor=CD2026&bcolor=F1F1F1). That meets WCAG AA requirements so I'm not too concerned. It might be worth trying a darker shade to see if you can hit AAA (7:1 ratio) for best readability, but that's up to you.
- On the link text for the bills and payments, I think I'm okay with the way they are (especially since Jim didn't object to them!) but this might be something to test with screen reader users when you head into research. Note that Sketch prototypes won't be accessible for testing, but there are alternative ways to create a screen reader-friendly prototype. The #inclusive-research channel in Slack is a great place to get resources related to this, and please feel to reach out to me directly too.

**Looking Ahead to Code:**
(Adding this section here for future reference later in the collaboration cycle! No immediate action required now.)
- With the links for bills and payments, we may want to find a way to associate the date and/or amount with the link text for screen reader users. My main concern is that a user might have multiple bills (eg. multiple unpaid copays at the same location), and right now those links wouldn't be distinguishable for screen reader users. There might be a few ways to do this and it will all depend on where the final design lands.
- In addition to the visible headings, I believe this page already has some visually hidden headings to help screen reader users navigate. When it's time to write code, we'll want to make sure the new subsections for bills and payments fit nicely into the hierarchy.

## Platform directions
- Update "Issue Title"
- Link to collab cycle touchpoint meeting epic
- Add to the Feedback pipeline on the Platform Governance team board
- Add your feedback
- Add assignees based on collab cycle touchpoint
  - **Design Intent**: VFS designer, VFS PM (optional), yourself (optional)
  - **Midpoint Review**: VFS PM, yourself (optional)
  - **Staging Review QA Only**: VFS PM, yourself (optional)

**Comment by ajakabcin** [comment]
Work to update color contrasts and consider changes to how titles of debts, copays and payments are linked tracked in #40774

**Comment by aagosto90** [comment]
This update has been made in the latest designs for [desktop](https://www.sketch.com/s/9b0e6efc-423a-4354-9db3-ab2083d566c9/a/ELmbGe2) and [mobile](https://www.sketch.com/s/9b0e6efc-423a-4354-9db3-ab2083d566c9/a/jg20r4z).
